src/main/java/uk/gov/hmcts/ccd/domain/service/casedataaccesscontrol/RoleAssignmentService.java

```diff
@@ -73,11 +77,22 @@ public RoleAssignments getRoleAssignments(String userId) {
             .collect(Collectors.toList());
     }
 
+    private RoleAssignments getOrganisationRA(List<RoleAssignment> roleAssignmentsList) {
+        return RoleAssignments.builder().roleAssignments(roleAssignmentsList.stream()
+            .filter(this::isValidOrganisationRA)
+            .collect(Collectors.toList())).build();
+    }
+
     private boolean isValidRoleAssignment(RoleAssignment roleAssignment) {
         final boolean isCaseRoleType = roleAssignment.getRoleType().equals(RoleType.CASE.name());
         return roleAssignment.isNotExpiredRoleAssignment() && isCaseRoleType;
     }
 
+    private boolean isValidOrganisationRA(RoleAssignment roleAssignment) {
+        final boolean isOrgRole = roleAssignment.getRoleType().equals(RoleType.ORGANISATION.name());
```

Comment:
isValidOrganisation --> can we this name

Reply:
:)

Reply:
DONE